modules/release-notes/pages/relnotes.adoc

```diff
@@ -27,6 +27,17 @@ This release adds support for the following platforms:
 
 See xref:install:install-platforms.adoc[Supported Platforms] for the complete list of supported platforms.
 
+[#deprecated-features-and-platforms-710]
+== Deprecated Features and Platforms
+
+[#deprecated-and-removed-platforms-710]
+=== Deprecated and Removed Platforms
+
+The following platforms are deprecated and will be removed in a future release:
+
+* Red Hat Enterprise Linux (RHEL) 7.x
+* CentOS 7.x
+
```

Comment:
Sorry one got missed from JIRA. 
Can we also add Ubuntu 18 as deprecated ?  per DOC-9785